config.go

```diff
@@ -30,6 +30,18 @@ func strEnvConfig(s *string, name string) {
 	}
 }
 
+func boolEnvConfig(b *bool, name string) {
+	*b = false
+	if env := os.Getenv(name); len(env) > 0 {
+		bVal, err := strconv.ParseBool(env)
+		if err != nil {
+			*b = false
+			return
+		}
+		*b = bVal
+	}
+}
```

Comment:
Redundant code for such simple action. This should be enough:

```go
if env, err := os.Getenv(name); err == nil {
  *b = env
}
```

It could be implemented even simpler if recognize set variable always as true, and unset one as false:

```go
if len(os.Getenv(name)) > 0 {
  *f = true
}
```

I prefer this way.

Reply:
I still insist that this should be changed.

Reply:
Well, commonly used envconfig uses `strconv.ParseBool`, so let's go this way. But it definitely should be implemented like this:

```go
if env, err := strconv.ParseBool(os.Getenv(name)); err == nil {
  *b = env
}
```

Reply:
Agreed :+1: